.gitignore

```diff
@@ -14,4 +14,3 @@ cmake_install.cmake
 /prboom2/prboom-plus
 /prboom2/prboom-plus-game-server
 /prboom2/prboom-plus.wad
-/prboom2/build*
```

Comment:
Why this?

Reply:
I added this some time ago by mistake when I was implementing `win_fopen.c`.

Reply:
Ah, okay.